config/locales/en.yml

```diff
@@ -190,6 +189,7 @@ en:
     address: Address
     admin: Admin
     affirmative: 'Yes'
+    assign: Assign
```

Comment:
A non-blocking question that just came to me: is it possible to store this in I18n key in a place where `<%= t('.assign') %>` would have worked?

I have no real idea if it is possible, hence the question.

Cheers!

Reply:
_maybe_ if we had a section for partials? im not sure if partials become "part of the page" in terms of the path for template strings or reconcile through the partial path.